apps/mobile/src/state/chat/hooks/useFetchNewMessages.ts

```diff
@@ -195,7 +195,7 @@ export default function useFetchMessagesForAllInboxes(): () => T.Task<'done'> {
             )()
         ),
         // @ts-expect-error bad typings?
```

Comment:
If there is not error here, can you please remove //@ts-expect-error ? If there still is an TS error, can you remove this change from changelog? We'll look at this in another MR